### DISCUSSION TITLE
`fallback` for `swrInifinite`

### DISCUSSION THREAD

**Discussion by shunkakinoki** [vercel/swr#1885] [Q&A]
is there anyway to do `fallback` for `useSWRInfinite` in root `SWRConfig` ?

if I want to pull `initialData` into from the`fallback` key of `SwrConfig` in the page root what is the correct key to put in it? `useSWRInfinite` seems to have a special  `$inf$`, attached to it.

https://swr.vercel.app/docs/with-nextjs

**Comment by stevenrein20** [comment]
I was looking for a way to do this too. I did not have any luck using SWRConfig. Instead I just passed the `initialData` into the page, then component to use it as the `options.fallbackData` on the `useSWRInfinite` hook. 
```
const { data, size, setSize } = useSWRInfinite(getKey, fetcher, {
    fallbackData: [initialData],
  });
```

Make sure that you pass the `initialData` as the only element in an array, this will keep data format consistent as you request more pages.

**Comment by koba04** [comment]
In this case, you have to use `unstable_serialize`  from `swr/infinite`.
https://github.com/vercel/swr/discussions/2164#discussioncomment-3668748
